docs/src/model_implementation.md

```diff
@@ -58,23 +58,22 @@ required_biogeochemical_auxiliary_fields(::NutrientPhytoplankton) = (:PAR, )
 biogeochemical_auxiliary_fields(bgc::NutrientPhytoplankton) = biogeochemical_auxiliary_fields(bgc.light_attenuation_model)
 ```
 
-Next, we need to define the methods that specify how the nutrient, ``N``, and phytoplankton ``P`` evolve. We want the phytoplankton to evolve at the rate given by:
-
+Next, we define the functions that specify how the phytoplankton ``P`` evolve. In the absence of advection and diffusion (which will be handled by Oceananigans), we want the phytoplankton to evolve at the rate given by:
 ```math
 \frac{\partial P}{\partial t} = \mu g(T) f(N) h(PAR) P - mP - bP^2,
 ```
 
-where ``\mu`` is `base_growth_rate`, ``m`` is `mortality_rate`, and ``b`` is `crowding`. The functions ``g``, ``f``, and ``h`` are given by:
+where $\mu$ corresponds to the parameter `base_growth_rate`, ``m`` corresponds to the parameter `mortality_rate`, and ``b`` corresponds to the parameter `crowding_mortality_rate`. Here, the functions ``g``, ``f``, and ``h`` are defined by:
 ```math
 \begin{align}
 g(T) &= c_1\exp\left(-c_2|T - T_{opt}|\right),\\
 f(N) &= \frac{N}{k_N + N},\\
 h(PAR) &= \frac{PAR}{k_P + PAR},
 \end{align}
 ```
-where ``c_1`` is `temperature_coefficient`,  ``c_2`` is `temperature_exponent`, ``T_{opt}`` is `optimal_temperature`, ``k_N`` is `nutrient_half_saturation`, and ``k_P`` is `light_half_saturation`. Since this is a simple two variable model and the total concentration is conserved, ``\frac{\partial N}{\partial t} = - \frac{\partial P}{\partial t}``.
+where $c\_1$ corresponds to `temperature_coefficient`,  $c\_2$ corresponds to `temperature_exponent`, ``T_{opt}`` corresponds to `optimal_temperature`, ``k_N`` corresponds to `nutrient_half_saturation`, and ``k_P`` corresponds to `light_half_saturation`. 
```

Comment:
Does it still render symbolically without the dollar signs?  I made this change because it appeared as \mu and c_1, etc. in the preview of the .md file

Reply:
Yeap!
See https://oceanbiome.github.io/OceanBioME.jl/previews/PR133/model_implementation/

Reply:
(Github’s preview of md files might not work, but Documenter renders this all right!)

Reply:
Oh, ok, thanks for clarifying.  It was on github where it looked off, but you're right that it is rendering correctly on documenter.